cql3/statements/alter_keyspace_statement.cc

```diff
@@ -81,33 +86,80 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
 #endif
 }
 
+static logging::logger mylogger("alter_keyspace");
```

Comment:
let's just put it at the top of the file as we do pretty much everywhere else, so it's made available to future changes without the need to define it upwards every time a new log statement needs it above the definition line.

Reply:
Ok.